src/forms/Dropzone.stories.tsx

```diff
@@ -9,44 +8,30 @@ export default {
 }
 
 export const defaultDropzone = () => {
-  const [progressValue, setProgressValue] = React.useState(0)
-  const [cloudinaryImage, setCloudinaryImage] = React.useState("")
-  const cloudName = text("Cloudinary Cloud", "")
-  const uploadPreset = text("Upload Preset", "")
-
-  const exampleUploader = (file: File) => {
-    CloudinaryUpload({
-      file: file,
-      onUploadProgress: (progress) => {
-        setProgressValue(progress)
-      },
-      cloudName,
-      uploadPreset,
-    }).then((response) => {
-      setProgressValue(100)
-      const imgUrl = `https://res.cloudinary.com/${cloudName}/image/upload/w_400,c_limit,q_65/${response.data.public_id}.jpg`
-      setCloudinaryImage(imgUrl)
-    })
-  }
-
   return (
     <>
       <Dropzone
         id="test-uploading"
         label="Upload File"
         helptext="Select JPEG or PNG files"
-        uploader={exampleUploader}
+        uploader={() => {}}
```

Comment:
Do you think it would be worth adding some user feedback here? Something like an alert indicating that "{file name} upload received." Could also see the case where a consumer might think the component always does that so up to you!

Reply:
Sure thing! Added an alert here